t/plugin/redirect.t

```diff
@@ -582,13 +582,48 @@ Location: https://foo.com:9443/hello
 GET /t
 --- error_code: 400
 --- response_body eval
-qr/error_msg":"failed to check the configuration of plugin redirect err: value should match only one schema, but matches both schemas 1 and 3/
+qr/error_msg":"failed to check the configuration of plugin redirect err: allOf 1 failed: value should match only one schema, but matches both schemas 1 and 3/
 --- no_error_log
 [error]
 
 
 
-=== TEST 26: enable http_to_https with upstream
+=== TEST 26: wrong configure, enable http_to_https with append_query_string
```

Comment:
Let's change the test case position to the end of the file (so there won't' have many test number changes).

Reply:
Done